dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/GetCartTest.php

```diff
@@ -0,0 +1,171 @@
+<?php /** @noinspection SpellCheckingInspection */
```

Comment:
Looks like we need to remove this string

Reply:
Thanks, it's PHPStorm stuff. Removed :)